### ISSUE TITLE
The second part of the README does not work in Sage 8.2

### ISSUE THREAD

**Issue by saraedum** [MCLF/mclf#3]
```
sage: from mclf import *
sage: sage: K = QQ
....: sage: v_3 = K.valuation(3)
....: sage: R.<x> = K[]
....: sage: Y = Superp(x^4+1, v_3, 3)
....: sage: Y
....: 
superelliptic curve Y: y^3 = x^4 + 1 over Rational Field, with respect to 3-adic valuation
sage: Y.conductor_exponent()
ValueError: The valuation [ Gauss valuation induced by (x)-adic valuation, v(y) = 1/2 ] does not approximate a unique extension of (x)-adic valuation with respect to y^3 + x*y + 2*x^2
```

**Comment by saraedum** [comment]
Fixed in https://trac.sagemath.org/ticket/25294. @swewers could you review that change?

**Comment by saraedum** [comment]
Will be fixed in 8.3.
